### PR TITLE
21806: Fixes react_series output detail formats

### PR DESCRIPTION
**Pull request by howsoRes**

### CODE REVIEW

File: howso/details.amlg [hunk 1]
```diff
@@ -830,6 +830,10 @@
 				(list)
 			)
 		)
-		(zip details)
+		(remove
```
Comment (howsohazard) [inline howso/details.amlg:836]: Why is this here?  Seems that either the method needs a comment or something.

File: howso/distances.amlg [hunk 2]
```diff
@@ -149,7 +149,7 @@
 		(accum (assoc
 			output
 				(assoc
-					distance_ratio
+					"distance_ratio"
```
Comment (howsohazard) [inline howso/distances.amlg:152]: why the quotes?

Comment (howsohazard) [reply howso/distances.amlg:152]: same for the changes below

File: howso/react_series_utilities.amlg [hunk 3]
```diff
@@ -179,7 +179,11 @@
 									(map (lambda (get (current_value) detail)) series_output)
 								)
 							)
-							(zip requested_details)
+							(remove
```
Comment (howsohazard) [inline howso/react_series_utilities.amlg:183]: This again seems weird.

File: howso/distances.amlg (hunk 2 above)
Comment (howsoRes) [reply howso/distances.amlg:152]: the quotes are because this is an output dictionary for the user, not declaring variables on the stack.  That's my preferred method of denoting the difference, an assoc with non-quoted keys are variables, quoted keys are simply keys for some data. Also it makes it consistent with the other detail outputs, this code was not consistent before. same for the others.

**Review by howsohazard — APPROVED** [review]
